docs-library/Template-for-10x-Investigation-Phase-One-Yes-Report.md

```diff
@@ -0,0 +1,42 @@
+
+## Recommendation
+
+* Do you recommend moving forward with this project?
+* What is the highest-value area for TTS to explore further? Why?
+## Why this matters
+* What is the background on this project?
+* What is the broad outline of the work that you want us to do in this area?
+* What type of impact could this project have, for TTS, for the government, and/or for the country?
+## How we arrived at this conclusion
+* What kind of research did you do to decide that this was where the work should focus?
+  * If you did market research, explain your methodology and sourcing, and what you learned from this research.
+  * If you did user research, explain who you spoke to and what you learned from those conversations.
+  * If you did other types of research, include information and conclusions drawn from that here as well.
+* Are there other departments in TTS or GSA, or other agencies, that we will need to partner with in order to move forward with this project?
+  * How will this project interact with those departments?
+  * What work has been done to ensure that we will be able to effectively align with those agencies around this project?
+* Are there existing or in-development projects that this work will need to align with? 
+  * How will this project interact with or complement the work of these other projects?
+  * What work has been done to ensure that we will be able to effectively align with those projects?
+## Proposal for the Discovery phase
+The next phase of 10x project work, Discovery, is intended to gain a detailed understanding about the industry, problem, product and market fit, finances, timeline, regulatory environment, how you scale, and so on. The team will think comprehensively about what could go right and wrong, and will create an initial strategy around each issue the project may face.
+
+* What is the plan for the discovery phase of work?
+  * This should be a narrative description of the overall work that needs to happen, not a project timeline or cost estimate
+* What are the differents type of work that the next phase will encompass?
+  * Understanding that things may change once the project is in process, please give as detailed a description of what you currently believe needs to be done next.
+## Questions the next phase should answer
+* What are the questions that you uncovered during this phase of work that will need to be answered in the Discovery phase?
+* What are the biggest unknowns facing the project? 
+* What are the uncertainties in the project plan that will need to get clarified in order to proceed effectively?
+* What are the possible risks associated with this project moving forward?
+## Draft Discovery Phase Staffing Plan
+
+[Link to staffing issue template](https://github.com/GSA/10x/blob/docs-library/docs-library/10x-Staffing-Issue-For-Phase-2.md)
```

Comment:
Note that in the URL it's currently routing to the `docs-library` branch – ultimately we'll need this to route to the `master` branch although those links won't work until we merge to master ... so kind of a chicken/egg problem. 

My recommendation would be to merge to master then go in and edit the URLs to be to the correct branch.